### DISCUSSION TITLE
custom jsx attribute names [Prettier Plugin]

### DISCUSSION THREAD

**Discussion by 0-don** [tailwindlabs/tailwindcss#7937] [Help]
I am using tailwindcss in react-native with the library [tailwind-rn](https://github.com/vadimdemedes/tailwind-rn), everything works fine and I get intellisense with the [extension ](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss)after adding custom class attribute in the extension settings ``style``, but I am having problems finding an option to add a custom class attribute in the [prettier-plugin-tailwindcss ](https://github.com/tailwindlabs/prettier-plugin-tailwindcss) package.

an example looks something like this:
```tsx
<View style={tailwind('bg-white relative rounded-md')}>
```

after looking at the code I've seen it checks for ``class`` and ``className``
https://github.com/tailwindlabs/prettier-plugin-tailwindcss/blob/f03d66cc4910b6ea2d1df81c7f9d285695362183/src/index.js#L318

do I need to fork the project and add it myself or am I missing something?

**Comment by AestasLonewolf** [comment]
Same issue here, very weird that [tailwindcss-intellisense](https://github.com/tailwindlabs/tailwindcss-intellisense) allows custom checks for custom attributes but [prettier-plugin-tailwindcss](https://github.com/tailwindlabs/prettier-plugin-tailwindcss) doesn't account for these

**Reply by 0-don** [reply]
I forked the project and added my own attribute names [here](https://github.com/Don-Cryptus/prettier-plugin-tailwindcss)
